### ISSUE TITLE
[ocl] CBF compression seg fault on travis

### ISSUE THREAD

**Issue by t20100** [silx-kit/silx#1644]
Occured on Travis with Python3.6 and pyopencl 2018.1: 
```
test_encode_to_bytes (silx.opencl.codec.test.test_byte_offset.TestByteOffset)
Test byte offset compression to bytes ... INFO:silx.opencl.processing:0.000MB are needed on device: Intel(R) Xeon(R) CPU @ 2.60GHz,  which has 7843.156MB
INFO:silx.opencl.processing:Compiling file ['codec/byte_offset'] with options 
INFO:silx.opencl.codec.byte_offset:increase data input buffer size to 7376647
INFO:silx.opencl.codec.byte_offset:increase compressed buffer size to 51636529
INFO:silx.opencl.processing:
Profiling info for OpenCL ByteOffset
                                  copy data H -> D:	9.738ms
                                  compression scan:	117.187ms
________________________________________________________________________________
                              Total execution time:	126.925ms
/home/travis/.travis/job_stages: line 57:  5715 Bus error               (core dumped) python run_tests.py --installed -v
```
(See https://travis-ci.org/silx-kit/silx/jobs/344378243)

**Comment by t20100** [comment]
Reproduced locally, looks to be related to pyopencl v2018.1
Seg fault occurs in  the deletion of a pyopencl object (apparently at CPP level during a `__del__`)

**Comment by t20100** [comment]
Fixed by #1666
